Reference_Manual.md

```diff
@@ -0,0 +1,3223 @@
+# Hazelcast C++ Client Library 
+
+* [Introduction](#introduction)
+* [Resources](#resources)
+* [Release Notes](#release-notes)
+* [1. Getting Started](#1-getting-started)
+  * [1.1. Installing](#11-installing)
+    * [1.1.1. Requirements](#111-requirements)
```

Comment:
Better to put these requirements under the "installing from source" section. Package managers will enforce/install the requirements themselves.

Reply:
Changed.